project/src/main/java/com/google/sps/utilities/DatabaseWrapper.java

```diff
@@ -95,7 +91,11 @@ public void insertOrUpdateUser(User user) {
    *
    * @param event the event to be inserted or updated; event's ID field should not exist in DB
    */
-  public void insertorUpdateEvent(Event event) {
+  public void insertOrUpdateEvent(Event event) {
+    SpannerOptions options = SpannerOptions.newBuilder().build();
```

Comment:
Make sure to pull in SpannerTasks from master.

Reply:
ditto, will use correct changes in merge